netbox_branching/templates/netbox_branching/inc/share_button.html

```diff
@@ -0,0 +1,6 @@
+{% if active_branch %}
+    {% load i18n %}
+    <a href="{{ url }}?_branch={{ active_branch.schema_id }}" type="button" class="btn btn-outline-blue">
```

Comment:
`url` isn't passed as a context variable. The rendered link _does_ work as the browser affixes the query string to the current URL automatically, but in the interest of cleanliness we should either omit `url` or replace it with a call to the object's `get_absolute_url()` method.